vendor/gopkg.in/olebedev/go-duktape.v3/duk_minimal_printf.c

```diff
@@ -7,6 +7,10 @@
 #include <stddef.h>  /* size_t */
 #include <stdint.h>  /* SIZE_MAX */
 
+#ifndef SIZE_MAX
```

Comment:
If you upgrade vendored libraries, please use `govendor` so that the hash in `vendor.json` also gets updated.